SPECS/kernel/config

```diff
@@ -6685,8 +6685,7 @@ CONFIG_NFS_DEBUG=y
 CONFIG_NFS_DISABLE_UDP_SUPPORT=y
 # CONFIG_NFS_V4_2_READ_PLUS is not set
 CONFIG_NFSD=m
-CONFIG_NFSD_V2_ACL=y
-CONFIG_NFSD_V3=y
+CONFIG_NFSD_V2=y
```

Comment:
`CONFIG_NFSD_V3` was removed as the option replaced NFSD_V2 as the default for `NFSD` in preparation of NFSD_V2 being depricated. We will continue to keep both on in Mariner 2.0

https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/10d55a17899a27100c6b1a6987e2ab6774c096a1
https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/a028a32ab61790fed7354f9a5ad2b7f0cdd11ec0